### ISSUE TITLE
Verify the getting-started prerequisite: setup.sh + download.sh from clean

### ISSUE THREAD

**Issue by kreuzhofer** [kreuzhofer/nebius-slurm-qwen38-demo#2]
## Question

Does the documented getting-started path work unmodified on this cluster, and what does it actually produce?

That path is exactly what any new user runs first, so it is verified as a standalone prerequisite before anything else on the map:

```
git clone ... && cd ... && bash scripts/setup.sh
source /mnt/data/qwen38-demo/activate.sh
bash /mnt/data/qwen38-demo/scripts/download.sh
```

`/mnt/data/qwen38-demo` does not exist yet, so this is a genuinely clean run.

Answer these concretely:

- Does `setup.sh` install the pinned stack (~3 GB of wheels) without resolver conflict, and does its closing GPU smoke test pass on a worker?
- Does the cu130 index ordering actually hold — is the installed torch `2.13.0+cu130`, not a PyPI build with no sm_103 kernels?
- Does `download.sh` fetch the model (~56 GB) and dataset, and what is the real on-disk size of each?
- What is `/mnt/data` free space **after** this completes? Charting measured 490 GB free of 2.0 TB, on a filesystem three other users write to.
- Does `activate.sh` exist and work from a fresh login shell?

Record any deviation from the README as a defect for the code-review ticket rather than fixing prose here.

**Comment by kreuzhofer** [comment]
## Resolution

**The documented getting-started path works unmodified.** `setup.sh` exited 0, its GPU smoke test passed on `worker-b300-1`, `activate.sh` works from a fresh shell, and `download.sh` fetched both artifacts. No code change was needed to get started — but several documented *figures* are wrong, listed below.

### GPU verification, verbatim

```
torch          : 2.13.0+cu130
cuda available : True
device         : NVIDIA B300 SXM6 AC
compute cap    : sm_103
matmul check   : True
transformers   : 5.16.1
peft           : 0.20.0
```

All nine pins installed at the exact requested versions (`torch 2.13.0+cu130`, `vllm 0.28.0`, `transformers 5.16.1`, `peft 0.20.0`, `datasets 4.6.0`, `accelerate 1.13.0`, `kernels 0.16.1`, `matplotlib 3.11.0`), with no resolver conflict. The cu130 index ordering held.

This is also direct confirmation of what [#4](https://github.com/kreuzhofer/nebius-slurm-qwen38-lora-demo/issues/4) found: since no published `torch==2.13.0` wheel ships sm_103 SASS, a working bf16 matmul on a cc-10.3 device means it is executing sm_100 code under Blackwell minor-version compatibility. The pins are sound; only the rationale written around them was wrong.

**`fla` and `fla_core` are both un-importable in the venv** — the empirical basis on which [#4](https://github.com/kreuzhofer/nebius-slurm-qwen38-lora-demo/issues/4) closed the sm_103 determinism risk. Re-check this after any dependency change.

### Disk, measured

| | size |
|---|---|
| venv | **8.1 GB** (README and `setup.sh` both say "~3GB of wheels") |
| model, 18 shards | **52 GB** (51.8 GiB — matches "~56GB across 18 shards") |
| dataset | 17 MB (README says ~50MB) |
| `/mnt/data` free: before → after | **490 GB → 430 GB** |

430 GB remaining is comfortable for the merged LoRA checkpoint (~54 GB) and the full fine-tune checkpoint (~54 GB) with room to spare, provided nobody else fills the volume.

### Dataset, measured

The raw `train` split holds **78,577** examples. The seed-42 95/5 split gives **74,649 train / 3,928 eval**, so one epoch at effective batch 128 is **583 steps** — which makes `train_full.py`'s "~583-step" docstring correct. The README's "74,659 train / 3,930 held out" is the post-split figure and accurate to about ten examples, but it never states the raw size, which invites exactly the misreading that produced a bogus 554-step figure earlier on this map (now corrected in [#5](https://github.com/kreuzhofer/nebius-slurm-qwen38-lora-demo/issues/5), [#11](https://github.com/kreuzhofer/nebius-slurm-qwen38-lora-demo/issues/11) and the map).

### Deviations recorded as defects, not fixed here

1. **The venv is 8.1 GB, not ~3 GB** — off by 2.7x, in both the README and `setup.sh`'s own progress message. On a shared 2 TB volume this is a budget claim worth getting right.
2. **`download.sh`'s disk-space guard is too low.** It warns below 130 GB, reasoning "56 GB base + 56 GB merged". The real full-path requirement is base 52 + merged 54 + full-FT 54 + venv 8 = ~168 GB, so the guard passes in situations that would later run out.
3. **Downloads are unauthenticated.** Both `hf download` and `load_dataset` warn `You are sending unauthenticated requests to the HF Hub... set a HF_TOKEN to enable higher rate limits and faster downloads`. Neither the README nor the script mentions `HF_TOKEN`; on a slower day this is the difference between a 5-minute and a much longer fetch.
4. **`hf download` contends on NFS locks.** The log is full of `Still waiting to acquire lock on .../.cache/huggingface/.gitignore.lock`. Waits were ~0.1 s each and harmless, but it is the second independent sign that this filesystem punishes concurrent small-file writes — the first being the `datasets.map()` Arrow-cache SIGBUS risk from [#3](https://github.com/kreuzhofer/nebius-slurm-qwen38-lora-demo/issues/3).
5. Cosmetic: the dataset is 17 MB on disk, not ~50 MB.

### Bonus: config-level architecture confirmation

`config.json` is now on disk and independently confirms much of what [#7](https://github.com/kreuzhofer/nebius-slurm-qwen38-lora-demo/issues/7) exists to check. Posted there rather than duplicated here.
